test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -208,7 +215,7 @@ std::string RandomCode::generate(int _maxOpNumber, RandomCodeOptions _options)
 	std::string code;
 
 	//random opCode amount
-	boostIntDistrib sizeDist (0, _maxOpNumber);
+	boostIntDistrib sizeDist (1, _maxOpNumber);
```

Comment:
we don't want to generate empty code. this is about how many opcode will be generated